tests/system_health/mellanox/mellanox_device_mocker.py

```diff
@@ -0,0 +1,131 @@
+from ..device_mocker import DeviceMocker
+from common.mellanox_data import SWITCH_MODELS
+from platform_tests.mellanox.mellanox_thermal_control_test_helper import MockerHelper, FanDrawerData, FanData, \
+    FAN_NAMING_RULE
```

Comment:
After PR #1864 is merged, the imports here must be:
```
from tests.common.mellanox_data import SWITCH_MODELS
from tests.platform_tests.mellanox.mellanox_thermal_control_test_helper import MockerHelper, FanDrawerData, FanData, \
    FAN_NAMING_RULE
```

Otherwise, it won't work.

Reply:
Fixed.